src/atoms/View/View.stories.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import View from './View';
+
+storiesOf('View', module)
+  .addParameters({
+    component: View,
+  })
+  .add('with text', () => <View>Simple Div</View>);
```

Comment:
```suggestion
  .add('with text', () => <View><Text>Simple View</Text></View>);
```
All text must be inside ```Text``` component. We need to be careful when we write stories since they are platform agnostic. This would break on native since ```View``` cannot contain text

Reply:
text web component is yet to be created.

Reply:
@susheelachoudry But the stories are common for react and react-native and hence they'll break. So create the text component as well before we merge this or raise a new PR to this branch for Text